sonic_installer/common.py

```diff
@@ -31,6 +31,8 @@ def run_command(command):
     if proc.returncode != 0:
         sys.exit(proc.returncode)
 
+    return out.rstrip("\n")
```

Comment:
>return out.rstrip("\n") [](http://example.com/codeflow?start=4&length=23)

To make coverage easier, do you really need this change?

Reply:
I need the return value on line 942.

Reply:
Since I will rollback line 942, I don't need this change any more.